File: Algorithms/Mathematics/Factorial/Interative/python/factorial.py
def factorial(n):
    fact = 1
    if n > 1:
        for i in range(2, n+1):
            fact *= i
    return fact

if __name__ == '__main__':
    n = int(input("Enter the number: "))
    print(f"Factorial of {n} is: {factorial(n)}")

File: Algorithms/Mathematics/Factorial/Interative/cpp/factorial.cpp
#include <iostream>

unsigned long factorial(unsigned int n) {
    unsigned long fact = 1;
    if(n > 1)
        for(int i=2; i<=n; i++)
            fact *= i;
    return fact;
}

int main() {
    std::cout << "Enter the number: ";
    unsigned int n;
    std::cin >> n;
    std::cout << "Factorial of " << n << " is: " << factorial(n) << std::endl;

    return 0;
}